docs/References.bib

```diff
@@ -35,6 +35,18 @@ @article{Balsara1999
   adsnote =      {Provided by the SAO/NASA Astrophysics Data System}
 }
 
+@article{Barkett2019uae,
+      author         = "Barkett, Kevin and Moxon, Jordan and Scheel, Mark A. and
```

Comment:
Please indent by 2 spaces to make consistent with the rest of this file, and follow the instructions linked at the top (https://spectre-code.org/writing_good_dox.html#writing_dox_citations). In particular, add `doi` and `url` fields.

Reply:
The file is already inconsistent. As I've said before, please provide a formatter if you want formatting to be consistent. I'll add that the formatting you started with is not what the Emacs formatter does, and so is more of an inconvenience than anything.

Reply:
Yes, basic formatting must be automated to keep the file consistent (#1381). But we should add DOIs and URLs, which we can't automate.

Reply:
I see now that this paper is not published in a journal yet, so it has no DOI, correct? Nvm my comment then :)

Reply:
Maybe I'm misremembering but I thought we discussed the possibility of using a python script to parse the bib file and check the fields for each. I'm not saying this needs to be done soon, just checking if I remember correctly :)

Reply:
We have a Python script that adds arXiv links to the documentation. For checking formatting and completeness of the entries we have no scripts yet. I suppose for basic formatting we would extent `FileTestDefs.sh` and for checking entries we could write a Python script.